packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/relational/DatabaseHelper.tsx

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Did I miss something, I thought we have something like this when transforming the execution plan? Also this file name should be `DatabaseEditorHelper` or something, it's conflicting with the other helper we have in `metamodels`